code-gen/src/main/scala/scalapb/zio_grpc/ZioCodeGenerator.scala

```diff
@@ -109,6 +109,8 @@ class ZioFilePrinter(
     private val accessorsClassName             = OuterObject / (service.name + "Accessors")
     private val accessorsWithMetadataClassName = OuterObject / (service.name + "WithMetadataAccessors")
 
+    private val CH = "_root_.scalapb.zio_grpc.server.ZServerCallHandler"
```

Comment:
The code style here is to use full names rather than acronyms, suggesting `zserverCallHandler` or `serverCallHandler`.